src/query/ast/src/ast/statements/view.rs

```diff
@@ -113,3 +114,58 @@ impl Display for DropViewStmt {
         )
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Drive, DriveMut)]
+pub struct ShowViewsStmt {
+    pub catalog: Option<Identifier>,
+    pub database: Option<Identifier>,
+    #[drive(skip)]
+    pub full: bool,
+    pub limit: Option<ShowLimit>,
+    #[drive(skip)]
+    pub with_history: bool,
```

Comment:
Why show view need with_history? show tables need it because drop table is not really drop. The table meta also in Meta server.

Reply:
O. I got you, we need this history. Please add some test about this.